packages/core/src/modules/non-secret/repository/NonSecretRecord.ts

```diff
@@ -0,0 +1,47 @@
+import type { RecordTags, TagsBase } from '../../../storage/BaseRecord'
+
+import { BaseRecord } from '../../../storage/BaseRecord'
+import { uuid } from '../../../utils/uuid'
+
+export type NonSecretRecordTags = TagsBase
+export type DefaultNonSecretRecordTags = {
+  connectionId: string
+}
+
+export type BasicMessageTags = RecordTags<NonSecretRecord>
+
+export interface NonSecretRecordStorageProps {
+  id?: string
+  createdAt?: Date
+  connectionId?: string
+  tags?: NonSecretRecordTags
+
+  content: string
```

Comment:
See the comment at the `NonSecretStorageService`. 

The interface `NonSecretStorageProps` would no take a generic as type of content. 

I am not too sure if this is the best option and would be happy to hear any arguments against it or alternatives.